doc/man3/OSSL_HTTP_transfer.pod

```diff
@@ -180,37 +180,39 @@ After disconnect the modified BIO will be deallocated using BIO_free_all().
 
 OSSL_HTTP_proxy_connect() may be used by an above BIO connect callback function
 to set up an SSL/TLS connection via an HTTPS proxy.
-It promotes the given BIO B<bio> representing a connection
+It promotes the given BIO I<bio> representing a connection
 pre-established with a TLS proxy using the HTTP CONNECT method,
-optionally using proxy client credentials B<proxyuser> and B<proxypass>,
-to connect with TLS protection ultimately to B<server> and B<port>.
-If the B<port> argument is NULL or the empty string it defaults to "443".
-The B<timeout> parameter is used as described above.
+optionally using proxy client credentials I<proxyuser> and I<proxypass>,
+to connect with TLS protection ultimately to I<server> and I<port>.
+If the I<port> argument is NULL or the empty string it defaults to "443".
+The I<timeout> parameter is used as described above.
 Since this function is typically called by applications such as
-L<openssl-s_client(1)> it uses the B<bio_err> and B<prog> parameters (unless
+L<openssl-s_client(1)> it uses the I<bio_err> and I<prog> parameters (unless
 NULL) to print additional diagnostic information in a user-oriented way.
 
-OSSL_HTTP_parse_url() parses its input string B<url> as a URL
+OSSL_HTTP_parse_url() parses its input string I<url> as a URL
 of the form C<[http[s]://]address[:port][/path]> and splits it up into host,
 port, and path components and a flag indicating whether it begins with 'https'.
 The host component may be a DNS name or an IP address
 where IPv6 addresses should be enclosed in square brackets C<[> and C<]>.
 The port component is optional and defaults to "443" for HTTPS, else "80".
-If the B<pport_num> argument is NULL the port specification
+If the I<pport_num> argument is NULL the port specification
 can be in mnemonic form such as "http" like with L<BIO_set_conn_port(3)>, else
 it must be in numerical form and its integer value is assigned to B<*pport_num>.
 The path component is also optional and defaults to "/".
-A path string returned is guaranteed to begin with a C</> character.
 On success the function assigns via each non-NULL result pointer argument
-the respective url component. On error, string results are assigned to NULL,
-else they are guaranteed to be non-NULL pointers to copies of the strings.
+I<phost>, I<pport>, I<pport_num>, I<ppath>, and I<pssl>
+the respective url component.
+On error, B<*phost>, B<*pport>, and B<*ppath> are assigned to NULL,
+else they are guaranteed to contain non-NULL string pointers.
 It is the reponsibility of the caller to free them using L<OPENSSL_free(3)>.
+A string returned via B<*ppath> is guaranteed to begin with a C</> character.
```

Comment:
Not sure why you are using B on the <*variable>

Reply:
I did this here (and elsewhere) to emphasize that this is not a parameter name but an expression involving a parameter name.